tests/routes/webapp/get_beneficiary_profile_test.py

```diff
@@ -106,6 +106,48 @@ def when_user_has_cancelled_some_offers(self, app):
                 {"domain": "physical", "current": 0.0, "max": 200.0},
             ]
 
+        @pytest.mark.usefixtures("db_session")
+        def when_user_is_created_without_postal_code(self, app):
+            # Given
+            booking = UserFactory(email="wallet_test@email.com", postalCode=None)
+            repository.save(booking)
+
+            # When
+            response = TestClient(app.test_client()).with_auth("wallet_test@email.com").get("/beneficiaries/current")
+
+            # Then
+            assert response.status_code == 200
+
+        @pytest.mark.usefixtures("db_session")
+        def when_user_is_a_pro(self, app):
+            # Given
+            user = UserFactory(
+                email="pro@example.com", postalCode=None, isBeneficiary=False, suspensionReason=None, dateOfBirth=None
+            )
+            user.suspensionReason = None
+            repository.save(user)
```

Comment:
```suggestion
            user = UserFactory(
                email="pro@example.com", postalCode=None, isBeneficiary=False, suspensionReason=None, dateOfBirth=None
            )
```

Tu mets déjà `suspensionReason` à None via la factory.

Reply:
Yes, mais pour une raison que je ne comprend pas (je n'ai pas creusé), la factory rajoute une valeur par défaut.
Quelques cas en prod ont le champ à None, du coup j'ai reproduit le comportement comme ça.

Tu as une idée d'où ça pourrait venir la valeur ajoutée par la factory ?